data/operators.csv

```diff
@@ -422,3 +422,4 @@ Parknet,ISP,signed + filtering,safe,197301,19561
 Kerfuffle,Cloud,signed + filtering,safe,35008,11481
 Rozint Ltd Co,ISP,signed + filtering,safe,21738,2442
 MilkyWan,ISP,signed + filtering,safe,2027,1829
+Telekom Malaysia Berhad,ISP,unsafe,4788,114
```

Comment:
Thanks for the PR!

The new line misses a field in the CSV file. See suggested change below.
```suggestion
Telekom Malaysia Berhad,ISP,,unsafe,4788,114
```